src/modules/Transcript/PriceChart/index.tsx

```diff
@@ -301,16 +246,117 @@ export function PriceChartUI(props: PriceChartUIProps): ReactElement {
 
 /** @notExported */
 export interface PriceChartProps extends PriceChartSharedProps {
+    eventId: string;
     headerExpanded: boolean;
 }
 
+const getPriceQuery = gql`
+    query QuotePrices($eventId: ID!, $after: DateTime) {
+        events(filter: { eventIds: [$eventId] }) {
+            id
+            quotePrices {
+                id
+                currentDayClosePrice
+                currentDayOpenPrice
+                endPrice
+                previousDayClosePrice
+                quote {
+                    id
+                    localTicker
+                    exchange {
+                        id
+                        shortName
+                    }
+                }
+                realtimePrices(after: $after) {
+                    id
+                    date
+                    price
+                    volume
+                    priceChangeFromStartValue
+                    priceChangeFromStartPercent
+                    volumeChangeFromStartValue
+                    volumeChangeFromStartPercent
+                    volumeChangeFromLastValue
+                    volumeChangeFromLastPercent
+                }
+                startPrice
+            }
+        }
+    }
+`;
+
+function generateChartData(eventId: string) {
```

Comment:
For consistency, do you mind renaming to `useChartData` since this is still a `hook`, just a custom one?

Reply:
👍